### PR TITLE
FELIX-6319 add configuration printer for framework properties

### PR DESCRIPTION
**Pull request by kwin**

### CODE REVIEW

File: webconsole/src/main/java/org/apache/felix/webconsole/internal/misc/FrameworkPropertiesPrinter.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    }
+
+    public void printConfiguration( PrintWriter printWriter )
+    {
+        getFrameworkProperties().entrySet().stream().forEach( e -> ConfigurationRender.infoLine( printWriter, null, e.getKey(), e.getValue() ));
+    }
+
+    private Map<String,String> getFrameworkProperties() 
+    {
+        Bundle systemBundle = getBundleContext().getBundle( Constants.SYSTEM_BUNDLE_LOCATION );
```
Comment (amitjoy) [inline webconsole/src/main/java/org/apache/felix/webconsole/internal/misc/FrameworkPropertiesPrinter.java:51]: You can also use `getBundle(Constants.SYSTEM_BUNDLE_ID)`

File: webconsole/src/main/java/org/apache/felix/webconsole/internal/misc/FrameworkPropertiesPrinter.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        BundleWiring bundleWiring = systemBundle.adapt(BundleWiring.class);
+        // https://docs.osgi.org/specification/osgi.core/7.0.0/framework.namespaces.html#framework.namespaces.osgi.native
+        Map<String, String> frameworkProperties = new TreeMap<>();
+        for ( BundleCapability capability : bundleWiring.getCapabilities( NativeNamespace.NATIVE_NAMESPACE ) )
```
Comment (amitjoy) [inline webconsole/src/main/java/org/apache/felix/webconsole/internal/misc/FrameworkPropertiesPrinter.java:55]: You can adapt the system bundle to `FrameworkDTO` and then retrieve the configured framework properties. That would be much easier this way.

File: webconsole/src/main/java/org/apache/felix/webconsole/internal/misc/FrameworkPropertiesPrinter.java (hunk 1 above)
Comment (kwin) [reply webconsole/src/main/java/org/apache/felix/webconsole/internal/misc/FrameworkPropertiesPrinter.java]: Done in https://github.com/apache/felix-dev/pull/104/commits/f7535ef846f5cb4e6ca9672116af7e2a1b0354ba

File: webconsole/src/main/java/org/apache/felix/webconsole/internal/misc/FrameworkPropertiesPrinter.java (hunk 2 above)
Comment (kwin) [reply webconsole/src/main/java/org/apache/felix/webconsole/internal/misc/FrameworkPropertiesPrinter.java]: Thanks for the hint, done in https://github.com/apache/felix-dev/pull/104/commits/f7535ef846f5cb4e6ca9672116af7e2a1b0354ba

**Comment by jbonofre** [comment]
@kwin so far so good, I'm doing couple of last tests and I will merge. Thanks !

**Review by jbonofre — APPROVED** [review]

**Pull request merged**